components/library/webkitgtk/patches/13-solaris-support.patch

```diff
@@ -55,7 +55,27 @@ diff --git a/Source/WTF/wtf/PlatformHave.h b/Source/WTF/wtf/PlatformHave.h
 index 41afbb4..9e5d453 100644
 --- a/Source/WTF/wtf/PlatformHave.h
 +++ b/Source/WTF/wtf/PlatformHave.h
-@@ -328,7 +328,7 @@
+@@ -229,16 +229,16 @@
+ #define HAVE_HOSTED_CORE_ANIMATION 1
+ #endif
+ 
+-#if OS(DARWIN) || OS(FUCHSIA) || ((OS(FREEBSD) || OS(NETBSD) || OS(OPENBSD) || OS(LINUX)) && (CPU(X86) || CPU(X86_64) || CPU(ARM) || CPU(ARM64) || CPU(MIPS) || CPU(RISCV64)))
++#if OS(DARWIN) || OS(FUCHSIA) || OS(SOLARIS) || ((OS(FREEBSD) || OS(NETBSD) || OS(OPENBSD) || OS(LINUX)) && (CPU(X86) || CPU(X86_64) || CPU(ARM) || CPU(ARM64) || CPU(MIPS) || CPU(RISCV64)))
+ #define HAVE_MACHINE_CONTEXT 1
```

Comment:
How this got defined previously (i.e. before the PR #15298 merge)?  Or, was it undefined?  If so, how the wasm compilation succeeded then?

Thank you.

Reply:
I have no experience building webkitgtk prior to 2.42.5.  If you are asking me to guess, then I suspect that wasm has architecture dependent code and did not get compiled in earlier OI builds.

Reply:
[This](https://hipster.openindiana.org/logs/oi-userland/10876/library.webkitgtk.publish.log) shows that wasm was built previously.  Check wins over guess :-).